front/src/api/index.tsx

```diff
@@ -1,7 +1,7 @@
-import axios from "axios";
-import { User } from "@fraud-ninja/shared-types";
+import axios from 'axios';
+import { User } from '@shared_types/user_types';
```

Comment:
The bet practice I think would still be to write an index in the shared_types that would export all the types at the same time instead of having to import the specifi type each time